apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/block-editor-nux.js

```diff
@@ -18,25 +19,37 @@ import VideoPressCelebrationModal from './video-celebration-modal';
 import WpcomNux from './welcome-modal/wpcom-nux';
 import LaunchWpcomWelcomeTour from './welcome-tour/tour-launch';
 
+const { unlock } = __dangerousOptInToUnstableAPIsOnlyForCoreModules(
```

Comment:
We needed to use `unlock` since the `select( 'core/edit-site' ).getCanvasMode` is still a private API.

- `getCanvasMode` selector: https://github.com/WordPress/gutenberg/blob/4b4b4ca96f201a06cf7fffb9921041957f070fac/packages/edit-site/src/store/private-selectors.js#L8
- It is used with `unlock` in Gutenberg; https://github.com/WordPress/gutenberg/blob/4b4b4ca96f201a06cf7fffb9921041957f070fac/packages/edit-site/src/components/sidebar-navigation-screen-template/index.js#L17

What do you think about using `@wordpress/private-apis`? @Automattic/lego 

If this is not preferred, I think we better wait until `getCanvasMode` get stable than implement this on our own.

Reply:
I assigned @Automattic/lego as a reviewer because I would like you to look at this earlier 🙂 If it looks good, I'll get to work on tests, etc.

Reply:
I assume it's good to use it except we have another stable way to get the canvas mode. Tested and changes look good 🙂

Reply:
I found these:

- Open issue [Separate Private APIs from Experimental APIs](https://github.com/WordPress/gutenberg/issues/47785)
- Coding guidelines [Use the lock() and unlock() API from @wordpress/private-apis to privately export almost anything](https://github.com/WordPress/gutenberg/blob/7fbf73d447b211c31d75585c46fb3f08e6267582/docs/contributors/code/coding-guidelines.md#use-the-lock-and-unlock-api-from-wordpressprivate-apis-to-privately-export-almost-anything)